types/kdbxweb/index.d.ts

```diff
@@ -0,0 +1,579 @@
+// Type definitions for kdbxweb 1.2
+// Project: https://github.com/keeweb/kdbxweb#readme
+// Definitions by: Roang-zero1 <https://github.com/Roang-zero1>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+export as namespace kdbxweb;
```

Comment:
The `kdbxweb` package doesn't seem to be a UMD module, so I don't think this line is needed.

Reply:
Yes that is true I have removed the namespace.